index.html

```diff
@@ -1,183 +1,294 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
     <!-- linking fonts -->
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <!-- linking styles -->
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles.css" />
+    <script src="script.js" defer></script>
     <title>Abdelhadi's Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <div class="navbar">
-            <a class="logo" href="#">my logo</a>
-            <nav>
-                <img class="burger" src="graphics/burger-menu.svg" alt="">
-                <ul>
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul>
-            </nav>
-        </div>
+      <div class="navbar">
+        <a class="logo" href="#">my logo</a>
+        <nav>
+          <img class="burger" src="graphics/burger-menu.svg" alt="">
+          <ul>
+            <li class="x"><img src="graphics/x.svg" alt="close"></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_171](https://user-images.githubusercontent.com/69638013/167598106-3ea5dbe9-7555-458a-85fb-adf19ca0e3c8.png)  |  ![Screenshot_172](https://user-images.githubusercontent.com/69638013/167597689-066c2e73-3f95-4d40-9c08-60dfcbb987ba.png)

- [ ] KIndly remove underline and use same font color used in the Figma design 👍🏿 
- [ ] Kindly use the same background color used in the Figma design 🤗 
- doing all these will make your design look more like the Figma design 👍🏿